### PR TITLE
Prepare for v3.1.0 release

### PR DESCRIPTION
**Pull request by felipecsl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,6 +254,22 @@ public class DeepLinkActivity extends Activity {
...
+```groovy
+tasks.withType(JavaCompile) {
+  options.compilerArgs << "-AdeepLinkDoc.output=${buildDir}/doc/deeplinks.txt"
+}
+```
+The documentation will be generated in the following format:
+```
+* {DeepLink1}\n|#|\n[Description part of javadoc]\n|#|\n{ClassName}#[MethodName]\n|##|\n
+* {DeepLink2}\n|#|\n[Description part of javadoc]\n|#|\n{ClassName}#[MethodName]\n|##|\n
+```
```
Comment (felipecsl) [inline README.md]: @jingwei423 @kxfang mind reviewing this to make sure it's accurate, please?

Comment (jingwei423) [inline README.md]: Looks accurate to me. I'm excited that it's getting released.

**Comment by felipecsl** [comment]
@jingwei423 v3.1.0 is already available on maven central!

**Comment by jingwei423** [comment]
@felipecsl awesome!

**Pull request merged**